data/json/npcs/talk_tags.json

```diff
@@ -1004,7 +1310,12 @@
       "What was that?",
       "Huh?  Is someone there?",
       "Who's there?",
-      "Who goes there?"
+      "Who goes there?",
+      "Hold up, did you just hear something?",
+      "Shh… what was that?",
+      "Wait, was that you talking just then?",
+      "<swear!>, you just hear someone talking?",
+      "That wasn't you just speaking, was it?  I think we have company."
```

Comment:
`That wasn't you, was it?`

Reply:
This might be better since it can allow for other noises within the context of the dialogue as opposed to just AI reacting to another AI's speech, but isn't that what this tag is for?